content/pipes.md

```diff
@@ -504,6 +504,11 @@ In addition to these, all `class-validator` options (inherited from the `Validat
     <td><code>boolean</code></td>
     <td>If set to true, validation errors will not be returned to the client.</td>
   </tr>
+  <tr>
+    <td><code>exceptionCode</code></td>
+    <td><code>number</code></td>
+    <td>Takes an HTTP status code and returns an exception object to be thrown. (Default: <code>400</code> or <code>HttpStatus.BAD_REQUEST</code>)</td>
```

Comment:
Without studying this in detail, it appears to me that this description describes a function, not a setting.  Does it actually "take a value" and "return an object", or does it just cause `class-validator` to behave this way?

Also (kind of a nit), but does this option belong inside a table described (and linked to) a list of `class-validator` options?  Don't have a perfect solution for that, but it might be confusing if the user looks for more info on this option that is "nest only" in the class validator docs.

Reply:
This setting allows you to specify which exception type will be used in case of an error (e.g., if you pass 404, it would call `throw new NotFoundException()`)

Reply:
Great.  I think we should adopt/adapt @kamilmysliwiec 's description above instead of the current one, which to me, reads like a description of a factory function.

Reply:
| Option                | Type     | Description|
| --------------------- | -------- | ---------- |
| `errorHttpStatusCode` | `number` | ​This setting allows you to specify which exception type will be used in case of an error (e.g., if you pass 404, it would call `throw new NotFoundException()`)|
| `errorHttpStatusCode` | `number` | ​This setting allows you to specify which exception type will be used in case of an error. By default, it would throw `new BadRequestException()`)|

@johnbiundo should we tell the user by default it will throw `BadRequestException()`? If it's not necessary, I will follow @kamilmysliwiec description.

Reply:
I like your second alternative.  

Suggestion: 

"By default, it would throw `new BadRequestException()`" 
-> 
"By default it throws `BadRequestException`"

Reply:
@johnbiundo it is done.

Reply:
one minor suggestion @Dominic-Preap. Can you add `.` (dot) at the end of the sentence? :) (to stay consistent with other rows)

Reply:
it is done. 👍